src/data/__tests__/getDataBy.ts

```diff
@@ -1,4 +1,4 @@
-import {getDataBy} from './getDataBy'
+import {getDataBy} from 'data/getDataBy'
```

Comment:
For the sake of consistency, I wanna keep test imports of the thing being imported as `../foo`
```suggestion
import {getDataBy} from '../getDataBy'
```